code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -253,7 +253,7 @@ a.popt {text-decoration: none;}
 .bold, .name, .prefix, .ooc, .looc, .adminooc, .admin, .medal, .yell {font-weight: bold;}
 
 /* ADD HERE FOR ITALIC */
-.italic, .italics,  .emote {font-style: italic;}
+.italic, .italics,  .emote {color: #6685f5;	font-style: italic;}
```

Comment:
don't do this.
italics class is for italics.

Reply:
Alright, I thought it wasn't used anywhere else but heard messages. I'll add a new hearing -span class in another PR and change all heard messages to that.